bin/options.js

```diff
@@ -75,16 +70,11 @@ const options = {
     group: SSL_GROUP,
     describe: 'HTTP/2, must be used with HTTPS',
   },
-  'static-directory': {
+  static: {
```

Comment:
Can be multiple

Reply:
@evilebottnawi Like `--static=example/dir1,example/dir2`?

Reply:
Like `--static=example/dir1 --static=example/dir2`